Taudem5PCVS2015/Taudem5PC.sln

```diff
@@ -63,6 +63,7 @@ Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "ogrtest", "ogrtest\ogrtest.
 EndProject
 Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "SinmapSI", "SinmapSI\SinmapSI.vcxproj", "{4C94C630-FAFC-4EC9-8639-336E6FD7A85C}"
```

Comment:
does this follow the new pattern developed for project/solution files in develop branch.